packages/main/src/SideNavigation.hbs

```diff
@@ -0,0 +1,71 @@
+<div class="ui5-sn-root">
+    {{#if items.length}}
+        <ui5-tree
+            id="ui5-sn-items-tree"
+            mode="SingleSelect"
+            ?_show-selected-on-sub-items="true"
+            @ui5-item-click="{{handleItemClick}}"
+        >
+            {{#each _currentItems}}
+                <ui5-tree-item
+                    icon="{{this.treeItem.icon}}"
+                    level="{{this.treeItem.level}}"
+                    .treeItem="{{this.treeItem}}"
+                    show-toggle-button-end="true"
+                    text="{{this.treeItem.text}}"
+                    ?has-children="{{this.treeItem.expandable}}"
+                    ?expanded="{{this.treeItem.expanded}}"
+                    ?_collapsed="{{this.collapsed}}"
+                    ?selected="{{this.treeItem.selected}}"
+                >
+                    {{#if ../_shoudlShowSubItems}}
```

Comment:
There is typo in "_shoudlShowSubItems" (should),  but moreover such variable does not exist at least I could not find it